otp/imx6ull-otp/imx6ull-otp.c

```diff
@@ -115,14 +116,14 @@ int otp_cnc_err(void)
 }
 
 
-void otp_wait(unsigned extra_fl)
+static void otp_wait(unsigned extra_fl)
 {
 	while (*(otp.base + ocotp_ctrl) & (OTP_BUSY | extra_fl)) usleep(10000);
 	usleep(100000);
 }
 
 
-int otp_write(unsigned addr, unsigned data)
+static int otp_write(unsigned addr, unsigned data)
```

Comment:
Handling dry run here would be simpler and more reliable - you missed dry running in `blow_boot_fuses`